ui/app/components/sidebar/nav/cluster.js

```diff
@@ -22,14 +22,15 @@ export default class SidebarNavClusterComponent extends Component {
     return this.namespace.inRootNamespace && !this.cluster?.hasChrootNamespace;
   }
 
-  get showSync() {
-    // Only show sync if cluster is not managed
-    return this.flags.managedNamespaceRoot === null;
-  }
+  get badgeText() {
+    const isHvdManaged = this.flags.isHvdManaged;
+    const onLicense = this.version.hasSecretsSync;
+    const isEnterprise = this.version.isEnterprise;
 
-  get syncBadge() {
-    if (this.version.isCommunity) return 'Enterprise';
-    if (!this.version.hasSecretsSync) return 'Premium';
-    return undefined;
+    if (isHvdManaged) return 'Plus';
```

Comment:
To make sure I understand correctly -- HVD vault clusters will show the "Plus" badge next to this nav item, whether or not their cluster is plus tier?

Reply:
Correct. The UI will cannot tell if the hvd cluster is plus or not, so we're basically upselling it to plus.